rotkehlchen/api/rest.py

```diff
@@ -3570,13 +3559,29 @@ def get_history_events(
                 cursor=cursor,
                 chain_id=chain_id,  # type: ignore
             )
+            ignored_ids_mapping = self.rotkehlchen.data.db.get_ignored_action_ids(
+                cursor=cursor,
+                action_type=ActionType.EVM_TRANSACTION,
```

Comment:
Shouldn't it be checking for all events and not only evm transactions?

Reply:
At the moment `ActionType` can only be

```
class ActionType(DBEnumMixIn):
    TRADE = 1
    ASSET_MOVEMENT = 2
    EVM_TRANSACTION = 3
    LEDGER_ACTION = 4
```

and the only history events that we currently ignore are the evm transactions. Ledger actions, trades and asset movements are not displayed in this view yet

Reply:
In a subsequent PR or release we probably need to change/remove this mapping. As we would need to ignore specific events. And since all of the above will become events the mapping makes no sense anymore.